projects/plugins/wpcomsh/tests/WpcomshTest.php

```diff
@@ -60,4 +60,20 @@ public function test_wpcomsh_make_content_clickable() {
 
 		$this->assertEquals( $expected_output, wpcomsh_make_content_clickable( $original_content ) );
 	}
+
+	/**
+	 * Tests if Jetpack Boost plugin is active, to test the integreation setup.
+	 *
+	 * This is for the `jp docker phpunit-integration` command to verify it works.
+	 *
+	 * @return void
+	 */
+	public function test_is_jetpack_boost_active() {
+		$plugins = getenv( 'JP_MONO_INTEGRATION_PLUGINS' );
+		if ( $plugins && strpos( $plugins, 'boost' ) !== false ) {
```

Comment:
It'd probably be a bit more robust to do like `strpos( ",$plugins,", ',boost,' ) !== false` to avoid the possibility that we someday have a "foo-for-boost" plugin or the like.

Or explode the string and check with `in_array()`.